### PR TITLE
Update to yml

### PR DESCRIPTION
**Pull request by PatrickAlphaC**
This should be a .yml and not a .md 

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 blank_issues_enabled: true
```
Comment (jkongie) [inline .github/ISSUE_TEMPLATE/config.yml:1]: You'll need to fix the indentation to pass the linting check.

```
blank_issues_enabled: true
contact_links:
  - name: Question
    url: https://stackoverflow.com/questions/tagged/chainlink
    about: Please ask and answer questions here.

```

Comment (PatrickAlphaC) [reply .github/ISSUE_TEMPLATE/config.yml:1]: Whoops, thank you. Ran `yarn prettier:run`

**Review by jkongie — APPROVED** [review]

**Comment by PatrickAlphaC** [comment]
@jkongie any idea on why CI failed here?

**Comment by jkongie** [comment]
Yeah we have a few flaky tests. You can just re-run the test

**Pull request merged**